README.md

```diff
@@ -387,6 +404,14 @@ An executable rule that pushes a Docker image to a Docker registry on `bazel run
       </td>
     </tr>
     <tr>
+      <td><code>format</code></td>
+      <td>
+        <p><code>Kind, required</code></p>
+        <p>The format as which to publish the image.  Currently, this supports
```

Comment:
"The format of the published image." or even
"The desired format of the published image."?